doc/modules/language-guide/pages/control-flow.adoc

```diff
@@ -196,17 +196,17 @@ The value `null` indicates that the iteration sequence has terminated.
 Until reaching `null`, each non-`null` value, of the form ``?``__n__ for some number _n_, contains the next successive element in the iteration sequence.
 
 [[intro-revrange]]
-== Using `revrange`
+== Using `revRange`
 
-Like `range`, the function `revrange` is a `class` that constructs iterators (each of type `Iter<Nat>`).
+Like `range`, the function `revRange` is a `class` that constructs iterators (each of type `Iter<Int>`).
```

Comment:
Hmm, I wonder why it's not symmetric with Range in motoko-base but at least this documents what we've got, strange as it is. Thanks for fixing.